### PR TITLE
ENG-6586 - Change  Access Level, add getScreen

### PR DESCRIPTION
**Pull request by kevinsites**

### CODE REVIEW

**Review by quacke-nid — APPROVED** [review]

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt [hunk 1]
```diff
@@ -203,28 +205,28 @@ class NeuroID private constructor(
...
-    fun getFirstTS(): Long = timestamp
+    internal fun getFirstTS(): Long = timestamp
 
-    fun getJsonPayLoad(context: Context): String {
+    internal fun getJsonPayLoad(context: Context): String {
         return getDataStoreInstance().getJsonPayload(context)
     }
 
-    fun resetJsonPayLoad() {
+    internal fun resetJsonPayLoad() {
```
Comment (kishisaka-nid) [inline NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:225]: We are using this in integration testing. Cannot make this one internal yet. Have to fix the test in our app first.

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt [hunk 2]
```diff
@@ -203,28 +205,28 @@ class NeuroID private constructor(
...
     }
 
-    fun getTabId(): String = NIDServiceTracker.rndmId
+    internal fun getTabId(): String = NIDServiceTracker.rndmId
 
-    fun getFirstTS(): Long = timestamp
+    internal fun getFirstTS(): Long = timestamp
 
-    fun getJsonPayLoad(context: Context): String {
+    internal fun getJsonPayLoad(context: Context): String {
```
Comment (kishisaka-nid) [inline NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:221]: this one also

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt [hunk 3]
```diff
@@ -203,28 +205,28 @@ class NeuroID private constructor(
...
         return getDataStoreInstance().getJsonPayload(context)
     }
 
-    fun resetJsonPayLoad() {
+    internal fun resetJsonPayLoad() {
         getDataStoreInstance().resetJsonPayload()
     }
 
-    fun captureEvent(eventName: String, tgs: String) {
+    internal fun captureEvent(eventName: String, tgs: String) {
```
Comment (kishisaka-nid) [inline NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:229]: same here

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt [hunk 4]
```diff
@@ -359,7 +355,7 @@ class NeuroID private constructor(
 
     fun isStopped() = NIDJobServiceManager.isStopped()
 
-    fun registerTarget(activity: Activity, view: View, addListener: Boolean) {
+    internal fun registerTarget(activity: Activity, view: View, addListener: Boolean) {
```
Comment (kishisaka-nid) [inline NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:358]: this one too.

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt (hunk 1 above)
Comment (kishisaka-nid) [reply NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:225]: removed from test, test fixed.

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt (hunk 3 above)
Comment (kishisaka-nid) [reply NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:229]: not needed in test, removed, test ok

File: NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt (hunk 2 above)
Comment (kishisaka-nid) [reply NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt:221]: not needed in test, removed, test ok

**Review by kishisaka-nid — APPROVED** [review]
updated, reviewed, tested, 🚢 it!

**Review by priya-xavier — APPROVED** [review]

**Pull request merged**